lib/OpenQA/Schema/Result/Bugs.pm

```diff
@@ -0,0 +1,182 @@
+# Copyright (C) 2017 SUSE Linux LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::Schema::Result::Bugs;
+use strict;
+use base 'DBIx::Class::Core';
+use Mojo::UserAgent;
+use OpenQA::Utils;
+use DBIx::Class::Timestamps 'now';
+
+__PACKAGE__->table('bugs');
+__PACKAGE__->load_components(qw(InflateColumn::DateTime Timestamps));
+__PACKAGE__->add_columns(
+    id => {
+        data_type         => 'integer',
+        is_auto_increment => 1,
+    },
+    bugid => {
+        data_type => 'text',
+    },
+    title => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    priority => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    assignee => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    status => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    resolution => {
+        data_type   => 'text',
+        is_nullable => 1,
+    },
+    bug_exists => {
+        data_type     => 'boolean',
+        default_value => 1,
+    },
+    t_last_refreshed => {
+        data_type   => 'timestamp',
+        is_nullable => 1,
+    },
+);
+__PACKAGE__->add_timestamps;
+__PACKAGE__->set_primary_key('id');
+__PACKAGE__->add_unique_constraint([qw(bugid)]);
+
+sub is_assigned {
+    my ($self) = @_;
+
+    if ($self->assignee eq 'None') {
+        return 0;
+    }
+    elsif ($self->assignee =~ m/\@forge\.provo\.novell\.com/) {
```

Comment:
no. The progress issue is about displaying the status of the issue/bug within the dashboard. For that we don't need to duplicate bugzilla - or worse: hardcode specific logic about novell assignees.

OpenQA's database can hold the informations necessary to pick the icon - but which icon to display belongs in SUSE specific scripts using some API. I don't want to see bugzilla fetching scripts within openQA. And I thought I made myself clear about that in the past.